src/reducers/sapCostSlice.ts

```diff
@@ -1,18 +1,22 @@
 import { IGroupSapCost, IProjectSapCost, ISapCost } from '@/interfaces/sapCostsInterfaces';
-import { getSapCosts } from '@/services/sapCostsServices';
+import { getSapCosts, getSapCurrentYear } from '@/services/sapCostsServices';
 import { RootState } from '@/store';
 import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
 import _ from 'lodash';
 
 interface ISapCostsState {
   projects: Record<string, IProjectSapCost>;
+  currentSap: Record<string, IProjectSapCost>;
   groups: Record<string, IGroupSapCost>;
+  currentSapGroups: Record<string, IGroupSapCost>;
```

Comment:
Could the names be something that describe the value better. For example currentYearSapProjects and currentYearSapGroups? currentSap could be confused with e.g. current sap connection.